search/search-domain/src/main/java/io/camunda/search/entities/RoleEntity.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record RoleEntity(Long roleKey, String name, Set<String> assignedUserKeys) {}
```

Comment:
🔧 sorry I realise I suggested that this could be a set of String but its actually `Long` :D